client/components/Archive/ArchiveItem.tsx

```diff
@@ -19,13 +19,15 @@ export const ArchiveItemComponent = ({item, priorities, urgencies, urgencyLabel}
         />
         <Column>
             <Row>
-                <UrgencyLabel
-                    item={item}
-                    label={urgencyLabel}
-                    urgencies={urgencies}
-                    tooltipFlow="down"
-                    inline={true}
-                />
+                {urgencies.length > 0 && (
+                    <UrgencyLabel
```

Comment:
would be good to handle it inside the component like we do in `PriorityLabel` so we don't have to wrap every usage